EntityLayer/Entities/Point.cs

```diff
@@ -10,5 +10,8 @@ public class Point
     {
         public double X { get; set; }
         public double Y { get; set; }
+        //public int ZoneId { get; set; }
```

Comment:
comment?

Reply:
This code will be used after creating the AssetZoneHistory class.